src/Workspaces/Core/Portable/Rename/ConflictEngine/ConflictResolver.cs

```diff
@@ -177,64 +178,76 @@ private static bool IsRenameValid(ConflictResolution conflictResolution, ISymbol
+                    if (otherThingsNamedTheSame.Any(s => s.IsKind(SymbolKind.Namespace)))
+                    {
+                        conflictingSymbolLocations = conflictingSymbolLocations.Concat(renamedSymbol);
+                    }
 
-                // There also might be language specific rules we need to include
-                var languageRenameService = project.LanguageServices.GetService<IRenameRewriterLanguageService>();
-                var languageConflicts = await languageRenameService.ComputeDeclarationConflictsAsync(
-                    conflictResolution.ReplacementText,
-                    renamedSymbol,
-                    renameSymbol,
-                    referencedSymbols,
-                    conflictResolution.OldSolution,
-                    conflictResolution.NewSolution,
-                    reverseMappedLocations,
-                    cancellationToken).ConfigureAwait(false);
+                    AddConflictingSymbolLocations(conflictingSymbolLocations, conflictResolution, reverseMappedLocations);
+                }
 
-                foreach (var languageConflict in languageConflicts)
+                // Some types of symbols (namespaces, cref stuff, etc) might not have ContainingAssemblies
+                if (renamedSymbol.ContainingAssembly != null)
                 {
-                    conflictResolution.AddOrReplaceRelatedLocation(new RelatedLocation(languageConflict.SourceSpan, conflictResolution.OldSolution.GetDocument(languageConflict.SourceTree).Id, RelatedLocationType.UnresolvableConflict));
+                    var project = conflictResolution.NewSolution.GetProject(renamedSymbol.ContainingAssembly, cancellationToken);
+
+                    // There also might be language specific rules we need to include
+                    var languageRenameService = project.LanguageServices.GetService<IRenameRewriterLanguageService>();
+                    var languageConflicts = await languageRenameService.ComputeDeclarationConflictsAsync(
+                        conflictResolution.ReplacementText,
+                        renamedSymbol,
+                        renameSymbol,
+                        referencedSymbols,
+                        conflictResolution.OldSolution,
+                        conflictResolution.NewSolution,
+                        reverseMappedLocations,
+                        cancellationToken).ConfigureAwait(false);
+
+                    foreach (var languageConflict in languageConflicts)
+                    {
+                        conflictResolution.AddOrReplaceRelatedLocation(new RelatedLocation(languageConflict.SourceSpan, conflictResolution.OldSolution.GetDocument(languageConflict.SourceTree).Id, RelatedLocationType.UnresolvableConflict));
+                    }
                 }
             }
+            catch (NullReferenceException e) when (FatalError.ReportUnlessCanceled(e))
```

Comment:
Should you just catch Exception?  That's our general pattern for this sort of thing, right?